frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import {
+  TextVariants,
+  Text,
+  TextContent,
+  Breadcrumb,
+  BreadcrumbItem,
+} from '@patternfly/react-core';
+import './create-storage-system.scss';
+
+export const CreateStorageSystemHeader: React.FC<CreateStorageSystemHeaderProps> = ({ url, t }) => (
```

Comment:
We should use `PageHeading` here.

Reply:
`PageHeading` is a very heavy component , and also not certain whether its aprt of sdk or not.(when I was coding this)
Plus, as per the recent changes, it would be the case we can drop this header when the new operand extension would come.

Reply:
I suggest we use PageHeading Operand Extension is going to take time. Page heading is going to be part of the SDK. As dashboards can't exist without `PageHeading`.

Reply:
`PageHeading` itself wont be required with the operand extension.